tests/results/test_state_result.py

```diff
@@ -121,9 +120,8 @@ async def test_graceful_retries_reraise_last_error_while_retrieving_missing_resu
     # the loop should have failed three times, sleeping 0.01s per error
     assert m.call_count == prefect.states.RESULT_READ_MAXIMUM_ATTEMPTS
     expected_sleep = (
-        prefect.states.RESULT_READ_MAXIMUM_ATTEMPTS
-        * prefect.states.RESULT_READ_RETRY_DELAY
```

Comment:
weirdly this test began failing; i guess removing these two fields resulted in actual performance gains? Anyways turns out the math was off by one here

Reply:
:doh: how was this passing before??  Thanks!

Reply:
Haha no worries, kind of funny that this one change was _just_ enough change something here.